backend/PostProcessing/IPostProcessing.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+#IPostProcessing.py
+#----------------------------------
+# Created By: Matthew Kastl
+#----------------------------------
+"""Post processing is responsible for preforming operation on data within the collection of data. 
+An example of this could be preforming an arithmetic operation on a column of data or cleaning missing values.)
+This module provides an interface to decouple this process from the main body of code. The factory method will 
+import and execute the correct code for a given process (based on a keyword) returning the result.
+New data will be inserted into the DataFrame by reference
+ """ 
+#----------------------------------
+# 
+#
+#Imports
+from abc import ABC, abstractmethod
+from importlib import import_module
+from pandas import DataFrame
+
+
+class IPostProcessing(ABC):
+
+    @abstractmethod
+    def post_process(self, data: DataFrame, **kwargs) -> bool:
+        raise NotImplementedError
+    
+
+def post_process_factory(data: DataFrame, key: str, **kwargs) -> bool:
+    """ Initiates a call to a class of IDataIngestion returning the result. The call is determined by a passed key, and arguments through the kwargs.
```

Comment:
Initiates a call to a class of IDataIngestion-> Initiates a call to a class of IPostProcessing?

Reply:
😅